docs/source/misc/changelog.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.1] - 2021-05-20
+
+### Added
```

Comment:
do you think we should detail this to any other part of the README or are we good?

Reply:
The neat thing about this is I've also added it here -> https://ml4ir.readthedocs.io/en/latest/misc/changelog.html

We can have a version of this in the main python README as well, but that would mean we will probably need to maintain two versions.